### PR TITLE
[Merged by Bors] - chore: update nightly-2022-04-04

### PR DESCRIPTION
**Pull request by EdAyers**

### CODE REVIEW

**Comment by EdAyers** [comment]
Currently failing because in `test/ext.lean` we get
[47](https://github.com/leanprover-community/mathlib4/runs/5823878449?check_suite_focus=true#step:6:47)
Error: index out of bounds

File: Mathlib/Mathport/Syntax.lean [hunk 1]
```diff
@@ -101,13 +101,13 @@ macro ak:Term.attrKind "notation3"
   for item in lits.getArgs do
     let lit := item[0]
     if let some _ := lit.isStrLit? then
-      macroArgs := macroArgs.push (← `(macroArg| $lit:strLit))
+      macroArgs := macroArgs.push (← `(macroArg| $lit))
```
Comment (Kha) [inline Mathlib/Mathport/Syntax.lean:104]: The new name of the antiquotation kind is `str`. There probably was a reason it was applied in the first place, which would explain the failure.

**Comment by EdAyers** [comment]
I think the `Error: index out of bounds` thing might have just been valid output

File: Mathlib/Mathport/Syntax.lean [hunk 2]
```diff
@@ -171,7 +171,8 @@ namespace Tactic
 /- E -/ syntax (name := mapply) "mapply " term : tactic
 /- M -/ syntax (name := withCases) "with_cases " tacticSeq : tactic
 syntax caseArg := binderIdent,+ (" :" (ppSpace (ident <|> "_"))+)?
-/- N -/ syntax (name := case') "case' " (("[" caseArg,* "]") <|> caseArg) " => " tacticSeq : tactic
+-- [todo] case' is now present in Lean core
+-- /- N -/ syntax (name := case') "case' " (("[" caseArg,* "]") <|> caseArg) " => " tacticSeq : tactic
```
Comment (digama0) [inline Mathlib/Mathport/Syntax.lean:175]:
```suggestion
-- [todo] case' is now present in Lean core
-- /- N -/ syntax (name := case') "case' " (("[" caseArg,* "]") <|> caseArg) " => " tacticSeq : tactic
```
```suggestion
/- N -/ syntax (name := case'') "case'' " (("[" caseArg,* "]") <|> caseArg) " => " tacticSeq : tactic
```
I very much doubt the one in lean core has exactly this syntax and behavior.

**Comment by gebner** [comment]
> I think the Error: index out of bounds thing might have just been valid output

IIRC, it's been there for a while.  I just didn't bother to track down the warning yet.

File: Mathlib/Mathport/Syntax.lean (hunk 2 above)
Comment (EdAyers) [reply Mathlib/Mathport/Syntax.lean:175]: What's the right thing to do in this case'?

Comment (digama0) [reply Mathlib/Mathport/Syntax.lean:175]: like my suggestion: just add more primes

**Comment by digama0** [comment]
bors merge

**Pull request closed without merging**